src/action_install.rs

```diff
@@ -121,6 +121,9 @@ fn install_all(
 		for (pkgbase, _depth, _split) in &packages {
 			let review_dir = rua_env.dirs.review_dir(pkgbase);
 			let build_dir = rua_env.dirs.build_dir(pkgbase);
+			let build_dir = build_dir
+				.canonicalize()
+				.unwrap_or_else(|e| panic!("Cannot canonicalize path {:?}, {}", build_dir, e));
```

Comment:
"Cannot canonicalize build_dir ..." ? E.g. being a bit more specific on where the canonicalization even happens?
Also, including the exact pkgbase would help - just in case?

Reply:
Although I better just do it myself, as easy as re-fetch everything, merge etc.